web_fix_modules_load/controllers/main.py

```diff
@@ -17,7 +17,7 @@ def _get_mod_names(self, mods):
         if isinstance(mods, str):
             mods = [int(x.strip()) for x in mods.split(",") if x.strip().isdigit()]
         if mods and isinstance(mods[0], int):
-            mod_names = model.browse(mods).mapped("name")
+            mod_names = ",".join(model.browse(mods).mapped("name"))
```

Comment:
@TDu are you sure about this change? We are supposed to get a list here. If you need it in `translations` just join there.

Reply:
With the module uninstalled, I see the module list being a comma separated string in the controllers. That is why I added this change.
But now I do not really know why both solutions seems to work the same :shrug:

Reply:
I finally removed this change